tensorflow/contrib/periodic_resample/__init__.py

```diff
@@ -0,0 +1,30 @@
+# =============================================================================
+# Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+"""Custom op used by periodic_resample."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow.contrib.periodic_resample.python.ops.periodic_resample_op import _periodic_resample_op
+from tensorflow.python.util.all_util import remove_undocumented
+
+periodic_resample =\
+    _periodic_resample_op.periodic_resample
```

Comment:
The best practices for custom ops are not well documented, sadly, but I think this is the problematic line. If you look at some code for `tf.contrib.tensor_forest`, you'll see that the `load_op_library()` call: https://github.com/tensorflow/tensorflow/blob/be79652c76a7cb82c4ea3afb8b9614233a22bbad/tensorflow/contrib/tensor_forest/python/ops/model_ops.py#L37 

...is always paired with an import of statically generated op wrappers:

https://github.com/tensorflow/tensorflow/blob/be79652c76a7cb82c4ea3afb8b9614233a22bbad/tensorflow/contrib/tensor_forest/python/ops/model_ops.py#L20

...and the statically generated op wrappers are always used in preference to the ones that might or might not be imported by a call to `load_op_library()`:

https://github.com/tensorflow/tensorflow/blob/be79652c76a7cb82c4ea3afb8b9614233a22bbad/tensorflow/contrib/tensor_forest/python/ops/model_ops.py#L63

The trick is to ensure that the statically generated model files are already created. You'll need a line like the following one:

https://github.com/tensorflow/tensorflow/blob/be79652c76a7cb82c4ea3afb8b9614233a22bbad/tensorflow/contrib/cmake/tf_python.cmake#L803

...and a matching `GENERATE_CONTRIB_OP_LIBRARY()` like the following:

https://github.com/tensorflow/tensorflow/blob/be79652c76a7cb82c4ea3afb8b9614233a22bbad/tensorflow/contrib/cmake/tf_core_ops.cmake#L99

Reply:
Thanks for the hints! I've made the changes. Hopefully this will work.